ruby-monday-blog/config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { registrations: 'users/registrations'}
```

Comment:
What do you think about using the newer hash syntax here? ` controllers: { registrations: 'users/registrations'}`